src/Components/dotnet-runtime-js/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "dotnet-runtime",
+  "private": true,
+  "description": ".NET is a developer platform with tools and libraries for building any type of app, including web, mobile, desktop, games, IoT, cloud, and microservices.",
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:dotnet/runtime.git"
+  },
+  "version": "1.0.0",
+  "main": "dotnet.js",
+  "types": "dotnet.d.ts",
+  "keywords": [
+    "dotnet",
+    "runtime",
+    "wasm"
+  ],
+  "author": "Microsoft",
+  "license": "MIT"
+}
```

Comment:
Moved the `@types/dotnet` code to its own "separate package" without deps because it was interfering with the workspace. In the future, we should ask for a regular npm package and consume `@types/dotnet` that way.